### DISCUSSION TITLE
Link to transcript

### DISCUSSION THREAD

**Discussion by UltraBelgGaming** [ItzDerock/discord-html-transcripts#48] [Q&A]
Hi there,
I was wondering if there is a way to get a link to the transcript that is created, for what I've seen you can only send the html file as an attachment?

**Comment by ItzDerock** [comment]
The package doesn't upload the transcript anywhere, so there is no link.
It would be like me trying to link you to a file I have in my clipboard -- I can't unless I upload that file somewhere.

**Comment by Zer0Cool0** [comment]
This can be achievable if you have a VPS with a web server installed. This VPS would also run the bot. You can easily download the html file to a location of your choice (ideally the folder where your webserver is) and you can access it via that link.
